dbLayer/src/test/java/com/plhal/ares/dblayer/EmbeddedPostgresTest.java

```diff
@@ -0,0 +1,55 @@
+package com.plhal.ares.dblayer;
+
+import io.zonky.test.db.AutoConfigureEmbeddedDatabase;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import javax.sql.DataSource;
+import java.util.ArrayList;
+import java.util.Optional;
+
+/**
+ * Class for application testing.
+ */
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = App.class)
+@AutoConfigureEmbeddedDatabase(beanName = "dataSource")
```

Comment:
Nedefinovat zadne zbytecne retezce. Ztezuje to udrzbu kodu (refaktoring). Navic v tomto pripade je to tu dle meho zbytecne.